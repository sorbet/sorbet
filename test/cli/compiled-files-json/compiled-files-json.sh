#!/usr/bin/env bash

set -euo pipefail

sorbet="$PWD/main/sorbet"

cd "$(dirname "${BASH_SOURCE[0]}")"

"$sorbet" --silence-dev-message --print compiled-files-json ./*.rb 2>&1
