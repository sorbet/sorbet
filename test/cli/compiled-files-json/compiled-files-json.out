No errors! Great job.
[{"file":"./compiled-false.rb","compiled":"false"},{"file":"./compiled-none.rb","compiled":"none"},{"file":"./compiled-true.rb","compiled":"true"}]